yarn-endpoints-behavior.html

```diff
@@ -102,6 +102,14 @@
       },
 
       /**
+       * The encoded request path built from the computed URL
+       */
+      path: {
+        type: 'String',
+        notify: true
```

Comment:
Mind making this `readOnly` so that it behaves like `url` (which is `readOnly` since it's computed)?